enumerable.rb

```diff
@@ -0,0 +1,25 @@
+module MyEnumerable
+  # all method
+  def all?
+    array1 = []
+    @list.each { |item| array1.push(item) if yield item }
+    puts array1.length == @list.length
+  end
```

Comment:
- [ ] Good job with your enumerables, but when testing the all method runs more than it should, check the example I tested it with two inputs and it gives the correct answer for example true but then it returns nil as well, kindly update it
![image](https://user-images.githubusercontent.com/62705947/209357439-ac62724b-8e12-44b8-84d7-08247fe3a226.png)
![image](https://user-images.githubusercontent.com/62705947/209357461-ec282f36-40d9-4ea9-acdc-d4a132fea8fb.png)